docs/api-reference.rst

```diff
@@ -2,6 +2,21 @@
 API Reference
 *************
 
+The API reference gives an overview of Geomstats implementation.
+
+The module `geometry` implements concepts in differential geometry,
+such as manifolds and Riemannian metrics, with associated exponential
+and logarithmic maps, geodesics, and parallel transport.
```

Comment:
I think this can be misleading, we don't implement manifolds themselves (as triangular meshes or I don't know how) but tools to compute with manifold-valued data.

Reply:
Done.